crates/pera-core/src/signature_mpc/mpc_events.rs

```diff
@@ -0,0 +1,77 @@
+use move_core_types::{ident_str, identifier::IdentStr, language_storage::StructTag};
+use pera_types::{
+    base_types::{ObjectID, PeraAddress},
+    id::ID,
+    PERA_SYSTEM_ADDRESS,
+};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+/// Generic trait for all MPC events.
+pub trait MPCEvent {
+    /// This function allows comparing this event with the Move event.
+    fn type_() -> StructTag;
```

Comment:
```suggestion
    fn tag() -> StructTag;
```
Generally I'm against using reserved names like `type`

Reply:
It's like that [anywhere in the codebase](https://github.com/dwallet-labs/dwallet-network/blob/1029338d4df75a053a51fc5891bbda6292203536/crates/pera-types/src/gas_coin.rs#L45), I think we'll better stick to their conventions